node/src/BaseClient.ts

```diff
@@ -7672,4 +7679,46 @@ export class BaseClient {
             throw err;
         }
     }
+
+    /**
+     * Reset the current connection password with a new password.
+     *
+     * This method is useful in scenarios where the server password has changed or when utilizing short-lived passwords for enhanced security.
+     * It allows the client to update its password to reconnect upon disconnection without the need to recreate the client instance.
+     * This ensures that the internal reconnection mechanism can handle reconnection seamlessly, preventing the loss of in-flight commands.
+     *
+     * This method updates the client's internal password configuration and does not perform password rotation on the server side.
+     *
+     * @param password - The new password to Reset the current password, or `null` to remove the current password.
+     * @param reAuth - If `true`, the client will re-authenticate immediately with the new password. If `false`, the new password will be used for the next connection attempt.
+     * @returns {Promise<GlideString>} A promise that resolves to the response from the server.
+     *
+     * @example
+     * ```typescript
+     * await client.resetConnectionPassword("newPassword", true) // "OK"
+     * ```
+     */
+    async resetConnectionPassword(password: string | null, reAuth: boolean) {
+        const resetConnectionPassword =
+            command_request.ResetConnectionPassword.create({
+                password: password,
+                reAuth,
+            });
+
+        const response = await this.createWritePromise<GlideString>(
+            resetConnectionPassword,
+        );
+
+        if (response === "OK" && !this.config?.credentials) {
```

Comment:
Why need to update it?

Reply:
It creates problems later, and it makes sense to do

Reply:
Please give more details. Other clients don't store config. Other clients don't store the config.

Reply:
well, you right. I addresed your question as why specifically password, but actually we dont have config at all. Odd to me.

Reply:
No, im wrong saying you right. We do save configuration. both in node and both in python. IDK about java. If i change the configuration i save it to the configuration. I dont create missalinged configuration between the core and the wrapper.